exercises/knapsack/description.md

```diff
@@ -1,18 +1,29 @@
 In this exercise, lets try to solve a classic problem.
 
-Bob is a thief. After months of carefuly planning, he finally manages to crack the security systems of a high-class apartment. 
+Bob is a thief. After months of careful planning, he finally manages to
+crack the security systems of a high-class apartment.
 
-In front of him are many items, each with a value (v) and weight (w). Bob of course wants to maximize the total value he can get; he would gladly take all of the items if he could. However, to his horror, he realized that the knapsack he carried with him can only hold so much weight(W).
+In front of him are many items, each with a value (v) and weight (w). Bob,
+of course, wants to maximize the total value he can get; he would gladly
+take all of the items if he could. However, to his horror, he realizes that
+the knapsack he carries with him can only hold so much weight (W).
 
-Given a knapsack with a specific capacity (W), help Bob determine the maximum value he can get from the items in the house. Note, Bob can only take only one of each item. 
+Given a knapsack with a specific carrying capacity (W), help Bob determine
+the maximum value he can get from the items in the house. Note that Bob can
+only take only one of each item.
 
-All values given will be strictly positive integers.  Items will be represented as a list of pairs, [wi, vi] where the first element `wi` is the weight of the ith item and `vi` is the value for that item.
+All values given will be strictly positive. Items will be represented as a
```

Comment:
Oh, I also changd "positive integers" to "positive", since this exercise may be implemented for languages that don't differentiate between integers and floating-point numbers.

Reply:
Yup no problem 👍